test/evpciph.txt

```diff
@@ -20,7 +20,6 @@
 #  xxd -ps -u
 
 Title = DES Tests (varous sources)
-
```

Comment:
oh yeah, good point.  i have them re-added, but will wait to see how the extended-test builds do first.